README.md

```diff
@@ -178,9 +178,16 @@ and exclude any alpha or beta versions.
 
 ## Release a New Live Demo Version
 
-Increment `currentVersion` in the gradle.properties file, commit, and push to the `main` branch.
-This will start a build on Jenkins, which will automatically create a tag. Creating the tag will start another build on Travis.
-The compiled, bundled, and ready to run `acrolinx-sidebar-demo-java.zip` will be uploaded to the release assets.
+Create a task branch, commit your changes.
+Increment `currentVersion` in the gradle.properties file.
```

Comment:
@nixor 

issue: Drop the SNAPSHOT while creating a release

Steps for releasing:

- Create a task branch.
- Drop the SNAPSHOT from gradle.properties file. example `0.11.1`
- Merge PR to main
- Create a tag via using the following commands (Or the GitHub UI)
  ```bash
  git tag v[major].[minor].[fix] (etc. v0.11.0)
  git push origin tag v[major].[minor].[fix]
  ```
- Github will create the specified tag on the remote repository and it will build and make a release when merged with the main branch there. Artifacts are automatically generated.

Once tag is created:

- Created new task branch
- Add new SNAPSHOT version. example: `0.12.0-SNAPSHOT`
- Create a PR and merge to main

Reply:
@abhijeetnarvekar Thanks adjusted.